mne/utils.py

```diff
@@ -381,7 +382,13 @@ def estimate_rank(data, tol=1e-4, return_singular=False,
         norms = _compute_row_norms(data)
         data /= norms[:, np.newaxis]
     s = linalg.svd(data, compute_uv=False, overwrite_a=True)
-    rank = np.sum(s >= tol)
+    if isinstance(tol, string_types):
+        if tol != 'auto':
+            raise ValueError('tol must be "auto" or float')
+        eps = np.finfo(float).eps
+        tol = np.max(data.shape) * np.amax(s) * eps
```

Comment:
But we changed the old values because they did not regularize enough. Should we check our PRNI paper script or the inverse cov examples before merging? Did you compare how this option behaves in critical situations?